ethereum/contracts/relayer/coreRelayer/CoreRelayer.sol

```diff
@@ -8,17 +8,9 @@ import "../../interfaces/relayer/IWormholeRelayerInternalStructs.sol";
 
 abstract contract CoreRelayer is CoreRelayerDelivery {
     /**
-     * @notice This 'send' function emits a wormhole message (VAA) that alerts the default wormhole relay provider to
-     * call the receiveWormholeMessage(DeliveryData memory deliveryData, bytes[] memory signedVaas) endpoint of the contract on chain 'targetChain' and address 'targetAddress'
-     * with the first argument being a DeliveryData struct with fields:
-     *      - sourceAddress: address (in wormhole 32-byte format) that called 'send' on the source chain
```

Comment:
It's bad practice to duplicate information too much in docs. These docs should describe the behavior of this function, not the entire system. Specifically, the user should read the docs on IWormholeReceiver.receiveWormholeMessages